Packs/MicrosoftGraphAPI/Integrations/MicrosoftGraphAPI/MicrosoftGraphAPI.py

```diff
@@ -141,8 +141,6 @@ def main() -> None:  # pragma: no cover
     app_secret = params.get('app_secret') or (params.get('credentials') or {}).get('password')
     certificate_thumbprint = params.get('certificate_thumbprint')
     private_key = params.get('private_key')
-    if not app_secret and not (certificate_thumbprint and private_key):
-        raise DemistoException('Application Secret or Certificate Thumbprint and Private Key must be provided.')
```

Comment:
In the [docs](https://github.com/demisto/content/pull/18493/files#diff-cc1ccabe6d29bfa8c195bc09c669eb91fbb5829e27aca6c4e43eba12c19532d9R27) we are saying that for self-deployed apps ***Application Secret*** and the ***Tenant ID*** should be filled so why are we removing this validation?

Reply:
For the self deployed, yes. 
TenentID and app secret. But this error jumps when customers are using the XSOAR app. 
The exception is wrong.

Reply:
I see, let's just verify with @DeanArbel who enhanced this part only a month ago